### PR TITLE
Update zh_CN.jsonc, zh_HK.jsonc and zh_TW.jsonc

### PR DESCRIPTION
**Pull request by BillKerman**
### Modifications:
1. Add terminologies for playlists and playlist folder in zh_CN,
2. Correct translation of _follow_, _playNext_, and _undoDislike_ according to the official translation on Apple Music in zh_CN, zh_HK and zh_TW,
3. Correct `discordRPC.cider`, `discordRPC.appleMusic` in zh_CN and zh_TW,
4. Correct `lastfmScrobble.removeFeatured` in zh_CN, zh_HK and zh_TW. **Featured** here doesn't mean recommendation, it means collaborators,
5. Add translation of `lastfmScrobble.filterLoop` and `spatial.notTurnedOn` in zh_CN,
6. Correct translation of playlist(播放清單) in zh_TW,
7. Correct some punctuation.

### CODE REVIEW

**Comment by maikirakiwi** [comment]
hold

**Comment by maikirakiwi** [comment]
IMG_3953

zh_TW on iOS does not use 播放清單 rather 播放列表

File: src/i18n/zh_TW.jsonc [hunk 1]
```diff
@@ -43,8 +43,13 @@
   "term.albums": "專輯",
   "term.artists": "藝人",
   "term.podcasts": "Podcasts",
-  "term.playlists": "播放列表",
-  "term.playlist": "播放列表",
+  "term.playlists": "播放清單",
```
Comment (maikirakiwi) [inline src/i18n/zh_TW.jsonc:46]: See comment

**Review by maikirakiwi — CHANGES_REQUESTED** [review]

File: src/i18n/zh_TW.jsonc (hunk 1 above)
Comment (BillKerman) [reply src/i18n/zh_TW.jsonc:46]: Snipaste_2022-02-10_01-39-58
Got it. I use iTunes translation here, probably deprecated.

**Pull request closed without merging**